src/app/modules/core/services/map.services.ts

```diff
@@ -1,87 +1,115 @@
-import { Injectable, ElementRef, EventEmitter, Injector, OnInit } from '@angular/core';
+import { Injectable, ElementRef, EventEmitter, Injector } from '@angular/core';
 import * as L from 'leaflet';
 import * as esri from 'esri-leaflet';
 import { HttpClient } from '@angular/common/http';
-
+import { Observable, of, Subject } from 'rxjs';
+import { map } from 'rxjs/operators';
+export interface layerControl {
+  baseLayers: Array<any>;
+  overlays: Array<any>
+}
 @Injectable()
-export class MapService implements OnInit {
+export class MapService {
 
-  public options: L.MapOptions;
+  public Options: L.MapOptions;
   // for layers that will show up in the leaflet control
-  public layersControl: any;
-  // for layers not shown in the leaflet control
-  //public ActiveLayers: Array<any> = [];
-
+  public LayersControl: Subject<layerControl> = new Subject<any>();
+  private _layersControl: layerControl = { baseLayers: [], overlays: [] };
   public CurrentZoomLevel;
-  public config;
-  public esriurl;
+  public CurrentLayer: String;
 
-  constructor(public http: HttpClient) {
+  constructor(http: HttpClient) {
 
-    this.options = {
-      layers: [
-        L.tileLayer('http://{s}.tile.openstreetmap.org/{z}/{x}/{y}.png', { })
-      ],
+    this.Options = {
       zoom: 10,
```

Comment:
lets go ahead and update this to be (39, -100, zoom level 4). I think this is center of the US